### PR TITLE
Update design of Payment page (hedvig-com)

### PR DESCRIPTION
**Pull request by robinandeer**
## What?

Update design of Payment page for Hedvig-com checkout flow.

## Why?

Match the rest of the experience with breadcrumbs and a more clear structure.

**Ticket(s): []**

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* master
  * **PR #1234** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1234" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1234?utm_source=stack-comment).

File: src/client/pages/Payment/Payment.tsx [hunk 1]
```diff
@@ -22,106 +25,162 @@ export const PaymentPage = () => {
...
 
-const PageHeader = styled.div({
-  height: '3.5rem',
-  paddingInline: '1rem',
-  display: 'flex',
-  justifyContent: 'space-between',
+const LargeParagraph = styled.p({
+  margin: 0,
+  fontSize: '1.5rem',
+  lineHeight: 1.26,
```
Comment (gustaveen) [inline src/client/pages/Payment/Payment.tsx:104]: nit: quite specific number :D did we mean 1.25?

**Review by gustaveen — APPROVED** [review]
LGTM! 👌👌

File: src/client/pages/Payment/Breadcrumbs.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { useTextKeys } from 'utils/textKeys'
+
+export const Breadcrumbs = () => {
+  const textKeys = useTextKeys()
+  return (
+    <Root>
+      <Step>{textKeys.BREADCRUMB_CHECKOUT()}</Step>
+      <Step active>{textKeys.BREADCRUMB_PAYMENT()}</Step>
```
Comment (guilhermespopolin) [inline src/client/pages/Payment/Breadcrumbs.tsx:10]: Don't we need to control which one is active? Part of another PR maybe?

**Review by guilhermespopolin — APPROVED** [review]

File: src/client/pages/Payment/Breadcrumbs.tsx (hunk 2 above)
Comment (robinandeer) [reply src/client/pages/Payment/Breadcrumbs.tsx:10]: The only option is the payment step -- we navigate back to racoon and a different page after success :)

File: src/client/pages/Payment/Payment.tsx (hunk 1 above)
Comment (robinandeer) [reply src/client/pages/Payment/Payment.tsx:104]: I picked it from Figma 😅

**Comment by robinandeer** [comment]
⏳ This pull request is set to merge as part of a Graphite merge job
Stack job ID: VKBNu2Ju5reyLkKf2vZ5.
See details on [graphite.dev](https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1234)

**Pull request merged**